packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -398,7 +399,9 @@ const TRANSFORMS = {
           version: version && parseXml(version).docker_version,
         }
       })(),
-      expNestedHvm: obj.platform['exp-nested-hvm'] === 'true',
+      expNestedHvm: semver.satisfies(obj.hardware_platform_version, '>=3.4')
+        ? obj.platform['nested-virt'] === 'true'
+        : obj.platform['exp-nested-hvm'] === true,
```

Comment:
IMHO, `expNestedHvm` should only return `obj.platform['exp-nested-hvm'] === ' true''` to be consistent with the XAPI and be flagged as `deprecated` with a small comment above explaining why, and what to use instead.
And maybe expose one property like: 
```js
isNestedVirtEnable: (isXcp83OrHigher && platformVersion >= 3.4 ? obj.platform['nested-virt'] : obj.platform['exp-nested-hvm']) === 'true' 
``` 

If you follow my suggestion, you should update the `BaseXoVm` type in `@vates/types/xo.mts` to add the new property and flag `expNestedHvm` as deprecated